docs/getting_started/basic_usage/data.rst

```diff
@@ -0,0 +1,55 @@
+.. _data:
+
+=================
+Data Input/Output
+=================
+
+The parameters that are used to correctly define the data inputs are the three dataset variables ``train``, ``dev`` and
+``eval``, as well as the parameter ``extern_data`` to define the data shapes. The dataset variables are set to a
+dictionary structure, where the key "``class``" defines which class implementation to load, and the other entries
+are passed as parameters to the constructor of the respective dataset implementation. A list of the available datasets
+can be found :ref:`here <dataset_reference>`.
+
+A very simple example would be:
+
+.. code-block:: python
+
+    train = {'class': 'HDFDataset', 'files': ['path/to/training_data.hdf']}
+
+Most datasets follow the convention that the input data is sequential and has the label "``data``", and the target data
+is sparse and has the label "``classes``". In the case of the hdf file it could be that the input data are
+100-dimensional MFCCs and the target data are 5,000 word classes. Then, ``extern_data`` would have to be defined as:
+
+.. code-block::
+
+    extern_data = {'data': (100, 2), 'classes': (5000, 1)}
```

Comment:
I'm not sure that's a good initial example for `extern_data`. That is the old format for `num_outputs`, which I generally would not recommend to use.